packages/map-popup/src/FocusTrapWrapper.tsx

```diff
@@ -0,0 +1,91 @@
+import React, { useCallback } from "react";
+
+/**
+ * Helper method to find the element within the app menu at the given offset
+ * (e.g. previous or next) relative to the specified element.
+ *
+ * @param {string} query  - Argument that gets passed to document.querySelectorAll
+ * @param {HTMLElement} element - Specified element (e.target)
+ * @param {1 | -1} offset - Determines direction to move within array of focusable elements (previous or next)
+ * @returns {HTMLElement} - element to be focused
+ */
+
+function getEntries(query: string) {
+  const entries = Array.from(document.querySelectorAll(query));
+  const firstElement = entries[0];
+  const lastElement = entries[entries.length - 1];
+
+  return { entries, firstElement, lastElement };
+}
+
+export function getNextSibling(
+  query: string,
+  element: EventTarget
+): HTMLElement {
+  const { entries, firstElement, lastElement } = getEntries(query);
+  const elementIndex = entries.indexOf(element as HTMLButtonElement);
+
+  if (element === lastElement) {
+    return firstElement as HTMLElement;
+  }
+  return entries[elementIndex + 1] as HTMLElement;
+}
+
+export function getPrevSibling(
+  query: string,
+  element: EventTarget
+): HTMLElement {
+  const { entries, firstElement, lastElement } = getEntries(query);
+  const elementIndex = entries.indexOf(element as HTMLButtonElement);
```

Comment:
`HTMLElement` might be more appropriate, since the entries depend on the query, and since we're returning these elements as HTMLElement on the next line.